Cherrypick fail test intermittently fails for windows

Cherrypick fail test intermittently fails for windows machine and it would be great if someone with a window machine can help us investigate this issue. 

```
: GET /api/stashes?path=C%3A%5CUsers%5Cappveyor%5CAppData%5CLocal%5CTemp%5C1%5Ctest-temp-dir11815-812-1m5qk8z.7wm9muz0k9&socketId=0
    1) cherrypick fail case
[server] 2018-02-05T22:26:33.357Z - info: POST /api/testing/cleanup
[server] stderr: 2018-02-05T22:26:33.360Z - debug: File change: testfile.txt
[server] 2018-02-05T22:26:33.360Z - info: testfile.txt triggered refresh for C:\Users\appveyor\AppData\Local\Temp\1\test-temp-dir11815-812-1m5qk8z.7wm9muz0k9
[server] stderr: 2018-02-05T22:26:33.360Z - debug: File change: testfile2.txt
[server] 2018-02-05T22:26:33.360Z - info: testfile2.txt triggered refresh for C:\Users\appveyor\AppData\Local\Temp\1\test-temp-dir11815-812-1m5qk8z.7wm9muz0k9
[server] stderr: 2018-02-05T22:26:33.361Z - debug: File change: .git\CHERRY_PICK_HEAD
[server] stderr: 2018-02-05T22:26:33.362Z - debug: File change: .git\COMMIT_EDITMSG
[server] stderr: 2018-02-05T22:26:33.362Z - debug: File change: .git\HEAD
[server] 2018-02-05T22:26:33.362Z - info: .git\HEAD triggered refresh for C:\Users\appveyor\AppData\Local\Temp\1\test-temp-dir11815-812-1m5qk8z.7wm9muz0k9
[server] stderr: 2018-02-05T22:26:33.362Z - debug: File change: .git\MERGE_MSG
[server] stderr: 2018-02-05T22:26:33.363Z - debug: File change: .git\config
[server] 2018-02-05T22:26:33.363Z - info: POST /api/testing/shutdown
[server] Stopped keeping ungit alive
UNGIT SERVER EXITED
  17 passing (1m)
  1 pending
  1 failing
  1) [BRANCHES]
       cherrypick fail case:
     Error: Timeout of 15000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
  
Warning: Task "mochaTest:click" failed. Use --force to continue.
Aborted due to warnings.
npm ERR! Test failed.  See above for more details.
```

--- a/nmclicktests/spec.branches.js
+++ b/nmclicktests/spec.branches.js
@@ -93,6 +93,7 @@ describe('[BRANCHES]', () => {
   it('cherrypick fail case', () => {
     return environment.nm.ug.click('[data-ta-clickable="node-clickable-0"]')
       .ug.click('[data-ta-action="cherry-pick"]:not([style*="display: none"]) .dropmask')
+      .wait(3000) // on windows clicking on cherry pick which results in conflicts might take some time
       .ug.click('.staging .btn-stg-abort')
       .ug.click('.modal-dialog .btn-primary')
       .wait(500)

